### ISSUE TITLE
[cscd-llh] compiler warning (Trac #2033)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#2025]
The indentation makes the code unclear.  Fix the indentation, or better yet actually use `{ }`.

```text
/home/bootcamp/icecube/src/cscd-llh/private/minimizer/I3CscdLlhSimplex.cxx: In
    member function ‘void I3CscdLlhSimplex::Get_psum(Mat_I_DP &, Vec_O_DP
        &)’:
/home/bootcamp/icecube/src/cscd-llh/private/minimizer/I3CscdLlhSimplex.cxx:372:5:
    warning: this ‘for’ clause does not guard... [-Wmisleading-indentation]

for (sum=0.0, i=0;i<mpts;i++)

^~~
/home/bootcamp/icecube/src/cscd-llh/private/minimizer/I3CscdLlhSimplex.cxx:374:7:
    note: ...this statement, but the latter is misleadingly indented as if it
    is guarded by the ‘for’
psum[j]=sum;
^~~~
```

Migrated from https://code.icecube.wisc.edu/ticket/2033
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:13:57", 
    "description": "The indentation makes the code unclear.  Fix the indentation, or better yet actually use `{ }`.\n\n{{{\n/home/bootcamp/icecube/src/cscd-llh/private/minimizer/I3CscdLlhSimplex.cxx: In\n    member function \u2018void I3CscdLlhSimplex::Get_psum(Mat_I_DP &, Vec_O_DP\n        &)\u2019:\n/home/bootcamp/icecube/src/cscd-llh/private/minimizer/I3CscdLlhSimplex.cxx:372:5:\n    warning: this \u2018for\u2019 clause does not guard... [-Wmisleading-indentation]\n\nfor (sum=0.0, i=0;i<mpts;i++)\n\n^~~\n/home/bootcamp/icecube/src/cscd-llh/private/minimizer/I3CscdLlhSimplex.cxx:374:7:\n    note: ...this statement, but the latter is misleadingly indented as if it\n    is guarded by the \u2018for\u2019\npsum[j]=sum;\n^~~~\n}}}", 
    "reporter": "david.schultz", 
    "cc": "aburgman", 
    "resolution": "fixed", 
    "_ts": "1550067237750774", 
    "component": "combo reconstruction", 
    "summary": "[cscd-llh] compiler warning", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2017-05-26T17:20:23", 
    "milestone": "", 
    "owner": "kjmeagher", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20170619T20:58:46: 
* olivas changed component from "analysis" to "icerec"
* olivas changed cc from "" to "aburgman"

**Comment by icecube-trac** [comment]
Trac update at 20170620T08:03:11: 
* kjmeagher commented:

> In changeset 156588:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="156588"
> Fix compiler warning assuming that the code currently functions correctly and that the indentation was not what the author intended. Fixes #2025
> ```


* kjmeagher changed resolution from "" to "fixed"
* kjmeagher changed status from "new" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:13:57: 
* olivas commented:

> Milestone June 2017 deleted


* olivas changed milestone from "June 2017" to ""
